src/Basis/Constructors.jl

```diff
@@ -99,23 +271,23 @@ function gaussquadrature(q::Int)
     end
     quadraturepoints[abs.(quadraturepoints).<10*eps()] .= 0
 
-    # return
-    return quadraturepoints, quadratureweights
+    return transformquadrature(quadraturepoints, quadratureweights, mapping)
 end
 
 """
 ```julia
-gausslobattoquadrature(q, weights)
+gausslobattoquadrature(q, weights, mapping)
 ```
 
-Construct a Gauss-Legendre-Lobatto quadrature
+Construct a Gauss-Legendre-Lobatto quadrature with the option of applying Conformal maps
```

Comment:
same question as above
```suggestion
Construct a Gauss-Legendre-Lobatto quadrature with the option of applying conformal maps
```